SAP/fundamental-react
# fix: [Datepicker] Scrolls to top of page on click

### Description
When opening a DatePicker component, the window scrolls to the top of the page.
This is because `Popper` initially renders at `top: 0` and `left: 0` and does not update focus after the popper.js has completed the positioning of the popper element.
See: https://github.com/popperjs/react-popper/issues/223

The solution is to perform a `window.scrollTo()` in order to counter-act the auto-scrolling.

fixes https://github.concur.com/coreui/nui-widgets/issues/834

To reproduce:
1. Open the docs page for DatePicker: https://sap.github.io/fundamental-react/?path=/docs/component-api-datepicker--primary
2. Click on a DatePicker component about half-way down the page. Note that it auto-scrolls to the top of the page with the component expanded

Testing Suggestions:
1. Start up the React playground docs, `npm start`
2. In the header at the top, click 'Docs'
3. Navigate to the 'DatePicker' page and click (or press key-down) on a component half-way down the page. It should no longer scroll to the top.
